environment-api/src/main/java/com/sequenceiq/environment/api/v1/proxy/model/response/ProxyResponse.java

```diff
@@ -56,4 +56,14 @@ public String getCreator() {
     public void setCreator(String creator) {
         this.creator = creator;
     }
+
+    @Override
+    public String toString() {
+        return super.toString() + ", " + "ProxyResponse{" +
+                "crn='" + crn + '\'' +
+                ", userName=" + userName +
+                ", password=" + password +
```

Comment:
please do not display the password

Reply:
+1 And please explicitly check for the absence of the password in the UT.